### PR TITLE
Upgrade to version 0.0.25

### PR DESCRIPTION
**Pull request by Leeoc**
- API version: `1.236.5`
- Package version: `0.0.25`

### CODE REVIEW

File: docs/PackageDependency.md
Comment (Leeoc) [inline docs/PackageDependency.md:1]: `PackageDependency` seems to have been replaced with `PackageDependencies`

Comment (jackgibson1) [reply docs/PackageDependency.md:1]: Any idea why this is?

Comment (Leeoc) [reply docs/PackageDependency.md:1]: IIRC it is linked to updates made to add SBOM dependencies into the dependencies endpoint

**Review by jackgibson1 — APPROVED** [review]
Nothing standing out to me here, although it may be good to get another set of 👀 before merging as I'm not fully familiar with this code!

**Review by AndrewSpeed — APPROVED** [review]

**Pull request merged**